src/main/java/com/commercetools/sync/internals/helpers/CustomHeaderSphereClientDecorator.java

```diff
@@ -43,7 +43,7 @@ protected HttpRequestIntent getHttpRequestIntent() {
         @Override
         public HttpRequestIntent httpRequestIntent() {
             final HttpHeaders headers =
-                    HttpHeaders.of(HttpHeaders.USER_AGENT, LIB_NAME + "( ver : " + LIB_VERSION + ")");
+                    HttpHeaders.of(HttpHeaders.USER_AGENT, LIB_NAME + " (ver : " + LIB_VERSION + ")");
```

Comment:
why not using `lib_name/lib_version`

Reply:
@ahmetoz 
To distinguish the change between old approach (adapt SolutionInfo) and new approach (decorator) after release.